app/_components/stations/Form.test.tsx

```diff
@@ -0,0 +1,117 @@
+import { render, screen } from '@testing-library/react';
+import Form from './Form';
+
+jest.mock('../inputs/Powers', () => ({
+  __esModule: true,
+  default: jest
+    .fn()
+    .mockReturnValue(
+      <input value="some value" onChange={(e) => e.target.value} />,
+    ),
+}));
+jest.mock('../inputs/Allegiances', () => ({
+  __esModule: true,
+  default: jest
+    .fn()
+    .mockReturnValue(
+      <input value="some value" onChange={(e) => e.target.value} />,
+    ),
+}));
+jest.mock('../inputs/Governments', () => ({
+  __esModule: true,
+  default: jest
+    .fn()
+    .mockReturnValue(
+      <input value="some value" onChange={(e) => e.target.value} />,
+    ),
+}));
```

Comment:
You can probably optimize this with something like 

```
const mockInputComponent = path => {
  jest.mock(path, () => ({
    __esModule: true,
    default: jest
      .fn()
      .mockReturnValue(
        <input value="some value" onChange={(e) => e.target.value} />,
      ),
  }));
};

const inputModules = [
  '../inputs/Powers',
  '../inputs/Allegiances',
  '...',
  '../inputs/Facilities',
];

inputModules.forEach(mockInputComponent);
```

Reply:
@nitishdhar Ive already tried this and I got the same errors as I did before I added the mocks, as if the mocking wasn't happening. Unfortunately I don't know Jest well enough to diagnose so any suggestions welcome - I've included my code below - otherwise I'd recommend not spending days on this and just refactoring later. The react-select tests all need work anyway

```
const mockInputComponent = (path: string) => {
  jest.mock(path, () => ({
    __esModule: true,
    default: jest
      .fn()
      .mockReturnValue(
        <input value="some value" onChange={(e) => e.target.value} />,
      ),
  }));
};

const inputPaths = [
  '../inputs/Powers',
  '../inputs/Allegiances',
  '../inputs/Governments',
  '../inputs/PowerEffects',
  '../inputs/FactionStates',
  '../inputs/Economies',
  '../inputs/LandingPads',
  '../inputs/StationTypes',
  '../inputs/Ships',
  '../inputs/Modules',
  '../inputs/Commodities',
  '../inputs/Facilities',
];

inputPaths.forEach((path) => mockInputComponent(path));
```

Reply:
Yeah, looks like there's some limitation in how jest hoists the mocks https://github.com/jestjs/jest/issues/11063. Let's leave it as is. Thanks for looking into this.